plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/ContractDescriptionConversionVisitor.kt

```diff
@@ -43,13 +47,12 @@ class ContractDescriptionConversionVisitor(
             .map { DuplicableCall(it) }
     }
 
-    fun getPostconditions(symbol: FirFunctionSymbol<*>, context: ContractVisitorContext): List<ExpEmbedding> {
-        return symbol.effects.map {
+    fun getPostconditions(context: ContractVisitorContext): List<ExpEmbedding> {
```

Comment:
It's a bit unfortunate that the calls for `getPreconditions` and `getPostconditions` do not match up, could you change that?